### PR TITLE
Removed SSReligion

### PR DESCRIPTION
**Pull request by ghost**
Removed a subsystem that served no purpose that global vars couldn't do better.

### CODE REVIEW

**Comment by QualityVan** [comment]
in this moment i am euphoric

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -62,10 +62,8 @@
 		strengthdiv = 8
 		for(var/mob/living/simple_animal/revenant/R in get_hearers_in_view(7,get_turf(holder.my_atom)))
 			var/deity
-			if(SSreligion.deity)
-				deity = SSreligion.deity
-			else
-				deity = "Christ"
+			if(GLOB.deity)
```
Comment (lzimann) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: why do you have the check if you are adding a `||` later

Comment (MoreRobustThanYou) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: woops!

**Comment by Anonmare** [comment]
🤠

**Comment by Cyberboss** [comment]
Because no one commented here why, subsystems act as a good way of namespacing global variables with zero overhead, so this is a step backwards rather than forwards

**Pull request closed without merging**